README.md

```diff
@@ -129,7 +129,7 @@ add_filter( 'wp_sentry_dsn', 'customize_sentry_dsn' );
 
 #### `wp_sentry_options` (array)
 
-You can use this filter to customize the Sentry options used to initialize the PHP tracker.
+You can use this filter to customize the [Sentry options](https://docs.sentry.io/error-reporting/configuration/) used to initialize the PHP tracker.
```

Comment:
```suggestion
You can use this filter to customize the Sentry [options](https://docs.sentry.io/clients/php/config/#available-settings) used to initialize the PHP tracker.
```

^ since we're not on the unified SDK yet that page has no useful information for the PHP integration

Reply:
You're right, fixed.